x-pack/test/ui_capabilities/common/services/ui_capabilities.ts

```diff
@@ -48,18 +50,29 @@ export class UICapabilitiesService {
     credentials?: BasicCredentials;
     spaceId?: string;
   }): Promise<GetUICapabilitiesResult> {
+    const features = await this.featureService.get();
+    const applications = Object.values(features)
+      .map(feature => feature.navLinkId)
+      .filter(link => !!link);
```

Comment:
The new capabilities API requires the front end to send the list of registered application. As this is supposed to be only a core internal calls, the actual list of application is not exposed. I had to find an alternative to get them for this test helper. Best way I found was to use the featureService to access the navLinks.

Reply:
do they always match 1:1?

Reply:
They do. `navLink` is the LP simili-equivalent to NP `application`. I wanted to rename `Capabilities.navLinks` to `Capabilities.applications` but that's some deep change in security and spaces capabilities modifiers, so I decided to postpone this to a separate PR.

Reply:
🙈

Reply:
🙈 ++